sklearn/metrics/_classification.py

```diff
@@ -1836,16 +1836,25 @@ def recall_score(
             This is applicable only if targets (``y_{true,pred}``) are binary.
         ``'micro'``:
             Calculate metrics globally by counting the total true positives,
-            false negatives and false positives.
+            false negatives and false positives (or summing their weights).
+            Gives same result as 'weighted'.
+            Except in multilabel classification, this and 'weighted' are both  
```

Comment:
```suggestion
            Except in multilabel classification, this and 'weighted' are both
```
A number of lines contain trailing whitespaces at the end. This generates a failure in the linting check.